src/corehost/cli/ijwhost/ijwhost.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
I don't really understand why we have `exports.cpp` and then this file... why not merge them into one?